tests/postman/newman/collection.json

```diff
@@ -17968,6 +17968,273 @@
             }
           ],
           "_postman_isSubFolder": true
+        },
+        {
+          "name": "Delete",
+          "item": [
+            {
+              "name": "Get wishlist data",
+              "event": [
+                {
+                  "listen": "test",
+                  "script": {
+                    "id": "ec93d088-5820-4e78-9f1b-d17f18cd7cee",
+                    "type": "text/javascript",
+                    "exec": [
+                      "const jsonData = pm.response.json();",
+                      "if (pm.environment.get('mage_type') === 'EE') {",
+                      "    pm.test(\"Status code is 200\", function () {",
+                      "        pm.response.to.have.status(200);",
+                      "    });",
+                      "    ",
+                      "    pm.test('Two wishlist exists', function () {",
+                      "        pm.expect(jsonData).length(2);",
+                      "    });",
+                      "}",
+                      ""
+                    ]
+                  }
+                }
+              ],
+              "request": {
+                "auth": {
+                  "type": "bearer",
+                  "bearer": [
+                    {
+                      "key": "token",
+                      "value": "{{customer-access-token}}",
+                      "type": "string"
+                    }
+                  ]
+                },
+                "method": "GET",
+                "header": [
+                  {
+                    "key": "Content-Type",
+                    "value": "application/json"
+                  }
+                ],
+                "body": {
+                  "mode": "raw",
+                  "raw": "{\n\t\"name\": \"{{seed}}-name\",\n\t\"visibility\": \"public\"\n}"
+                },
+                "url": {
+                  "raw": "{{domain}}{{endpoint_wishlists}}",
+                  "host": [
+                    "{{domain}}{{endpoint_wishlists}}"
+                  ]
+                }
+              },
+              "response": []
+            },
+            {
+              "name": "Delete wishlist 2",
+              "event": [
+                {
+                  "listen": "prerequest",
+                  "script": {
+                    "id": "6c2b232d-71bb-4653-b0a2-ba5b47722d7d",
+                    "type": "text/javascript",
+                    "exec": [
+                      "pm.environment.set(\"seed\", \"wishlist-\" + Math.floor(Math.random() * Math.floor(100)));"
```

Comment:
ur delete methods again contain a body and pre-request :P

Reply:
removed